code/modules/mob/camera/camera.dm

```diff
@@ -18,6 +18,7 @@
 /mob/camera/forceMove(atom/destination)
 	var/oldloc = loc
 	loc = destination
+	SEND_SIGNAL(src, COMSIG_MOVABLE_LOCATION_CHANGE, oldloc)
```

Comment:
Again, signals should only be being called from one place.

Can we abstract this whole behavior into a separate proc that they call, or find a way to get these to call `..`?

Reply:
i assume the reason theyre doing this is so that they dont call Crossed or Uncrossed or the other movement procs since its not supposed to exist to anything else in the world. I guess i can make an abstract_move proc for this behavior.

Reply:
update: created an abstract_move() proc for things like this, all overrides call parent which calls update_loc() (which is the only place that sends COMSIG_MOVABLE_LOCATION_CHANGE)